### ISSUE TITLE
Interactive tokens should be keyboard accessible

### ISSUE THREAD

**Issue by muratsu** [sourcegraph/sourcegraph#22952]
Currently, when we're navigating the source code on Sourcegraph the interactive tokens are not keyboard accessible. This is potentially an a11y compliance issue
